.github/workflows/manual_destroy.yml

```diff
@@ -0,0 +1,126 @@
+name: 'Delete Deployment Test Resources'
+
+on:
+  pull_request:
+    branches:
+      - main
+    types: [opened]
+  workflow_dispatch:
+    inputs:
+      resource:
+        description: "Resource from deployment tests to destroy"
+        required: true
+        type: choice
+        options:
+          - projects
+          - folders
+        default: 'projects'
+
+jobs:
+  destroy-folders:
+    name: "Destroy folder"
+    runs-on: ubuntu-latest
+    if: github.event_name == 'pull_request' ||  github.event.inputs.resource == 'folders'
```

Comment:
Should this be logical AND, not OR?

Reply:
No it should be OR because the `github.event.inputs` are only available on `workflow_dispatch`. Since this is run on either a pull request to main or run manually (will only be available after this is merged into main) it needs to be an or to run